android-ktx/src/main/java/com/mirego/trikot/viewModels/SwitchViewModelBinder.kt

```diff
@@ -26,6 +28,10 @@ object SwitchViewModelBinder {
                 switch.isChecked = it
             }
 
+            viewModel.enabled
+                .distinctUntilChanged()
```

Comment:
I don't think it's the platform job to distinct on these values. It should be done only on the ViewModel implementation.

Also, why is `.asLiveData()` required here while the viewModel.isOn on line 27 is not?

Reply:
Not gonna lie. I copied what was done for `ButtonViewModelBinder`

Reply:
If you look at the implementation of the extension fun `Publisher<T>.observe`:
```
fun <T> Publisher<T>.observe(lifecycleOwner: LifecycleOwner, observeBlock: ObserveBlock<T>) =
    asLiveData().observe(lifecycleOwner, observeBlock)
```
it is doing asLiveData() behind the scene